app/src/main/java/org/mozilla/fenix/tabstray/browser/InactiveTabsAdapter.kt

```diff
@@ -84,7 +90,7 @@ class InactiveTabsAdapter(
         val items = tabs.list.map { Item.Tab(it) }
         val footer = Item.Footer(context.autoCloseInterval)
 
-        submitList(listOf(Item.Header) + items + listOf(footer))
+        submitList(listOf(Item.Header) + items + listOf(Item.RecentlyClosed) + listOf(footer))
```

Comment:
nit:
```suggestion
        submitList(listOf(Item.Header) + items + listOf(Item.RecentlyClosed, footer))
```

You could hoist this list up to a variable if it's inconvenient to have it inlined; up to you. 🙂